posit_workbench_login_guidance.md

```diff
@@ -25,9 +25,9 @@
 
 https://github.com/Public-Health-Scotland/R-Resources/blob/master/posit_workbench_and_kubernetes.md#best-practice
 
-10. Once you have selected a session size click the "Start Session" button.
+10. Once you are satisfied with the details include and selected a session size, click the "Start Session" button.
```

Comment:
I think this needs reworded, it's a bit unclear

Reply:
Agreed. I've added more structure and another screenshot to help with this.